### PR TITLE
Start of new donation landing page

### PR DESCRIPTION
**Pull request by krbullock**
This adds the progress banner on `/support/`, ported over from salesforce-stripe (and updated slightly to move towards the new mockups). Let me know if there's a way to do it that integrates better with the WP stylesheets.

### CODE REVIEW

**Comment by jonathanstegall** [comment]
Note to self: this is part of issue #20 in this repo, and also https://github.com/MinnPost/minnpost-largo/issues/30 in the theme.

**Review by jonathanstegall — APPROVED** [review]
This works well. I have a few CSS changes to integrate more, but not many!

File: templates/front-end/support.php [hunk 1]
```diff
@@ -11,7 +11,31 @@
 ?>
 
 	<div id="primary" class="m-layout-membership o-support">
+
 		<main id="main" class="site-main" role="main">
+
+			<div class="m-support-progress" aria-label="progress">
```
Comment (krbullock) [inline templates/front-end/support.php]: On the Flask app, the progress bar is outside the `<main>` content, which I _think_ aligns with the intended use of the `<main>` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#Usage_notes

...thoughts?

Comment (jonathanstegall) [reply templates/front-end/support.php]: You're totally right. Probably is what my previous self was thinking. I'll restore that.

**Pull request merged**